File: Weather-server/Models/Client/WeatherModel.cs
namespace Weather_server.Models.Client
{
    public class WeatherModel
    {
        public Guid Id { get; set; }
        public DateTime Date { get; set; }
        public float Temperature { get; set; }
        public float Humidity { get; set; }
        public float Td { get; set; }
        public int Pressure { get; set; }
        public string WindDirection { get; set; } = string.Empty;
        public int WindSpeed { get; set; }
        public float Cloudy { get; set; }
        public int H { get; set; }
        public int VV { get; set; }
        public string WeatherCondition { get; set; } = string.Empty;
    }
}

File: Weather-client/src/app/models/Weather.model.ts
export interface IWeather {
    id?: string;
    date: Date;
    temperature: number;
    humidity: number;
    td: number;
    pressure: number;
    windDirection: string;
    windSpeed: number;
    cloudy: number;
    h: number;
    vv: number;
    weatherCondition: string;
}